### ISSUE TITLE
Website Query Tool

### ISSUE THREAD

**Issue by austin-berry** [EniariGames/StaticShock#6]
Develop .PHP on website to return sql data results on a posted ID to the server. Make sure there is a proper format or standard format that can easily be converted to game type variables.

**Comment by austin-berry** [comment]
Since the servers are currently down at Jayson's, we can hold off on this project item for now. We need to get on this as soon as everything is back online though. Angel, you said you have most of it completed already?

**Comment by Alcatraz7** [comment]
Yes, most of it is ready. Just need to test it and make sure I am getting the correct output.

**Comment by austin-berry** [comment]
@Alcatraz7 Status update?
